automatic/eid-belgium/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,18 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName    = 'eid-belgium'
+  fileType       = 'msi'
+  softwareName   = 'eid-belgium*'
```

Comment:
this doesn't seem to be correct, is this actually what is shown in `Add/Remove programs` in the control panel?

Reply:
You were right. This was `Belgium e-ID` with a long suffix. Taken care off in https://github.com/wget/chocolatey-package-eid-belgium